src/components/course/data/courseLoader.js

```diff
@@ -64,7 +62,21 @@ export default function makeCourseLoader(queryClient) {
       queryClient.ensureQueryData(queryCouponCodeRequests(enterpriseCustomer.uuid, authenticatedUser.email)),
       queryClient.ensureQueryData(queryBrowseAndRequestConfiguration(enterpriseCustomer.uuid)),
     ]);
-
+    const redeemableLearnerCreditPoliciesLoader = await queryClient.ensureQueryData(queryRedeemablePolicies({
```

Comment:
nit/question: any particular reason this couldn't be named `redeemableLearnerCreditPolicies`, given `await queryClient.ensureQueryData` is returned the _value_, not a loader-esque function

Reply:
Ah, I see. Might it make sense to remove the `queryRedeemablePolicies` from the `subsidyQueries` list such that the code path that relies on the `redeemableLearnerCreditPolicies` returned here, instead of extracting the response data from the `subsidyQueries` list? That way, all policy data comes from a single source throughout the loader.

If we did that, might make sense to also possibly rename `subsidyQueries` to `otherSubsidyQueries` (since we're abstracting one of the subsidy-specific queries from the list; it's no longer exhaustive of _all_ subsidy queries).